lib/handle-pull-request-change.js

```diff
@@ -28,7 +27,7 @@ async function commitsAreSemantic (commits, scopes, types, allCommits = false, a
 
 async function handlePullRequestChange (context) {
   const { title, head } = context.payload.pull_request
-  const userConfig = await getConfig(context, 'semantic.yml', {})
+  const userConfig = await context.config('semantic.yml')
```

Comment:
To use this probot has to updated to at least v9.3.0, which might require some more changes.